test/y2storage/boot_requirements_checker_x86_test.rb

```diff
@@ -290,14 +290,31 @@
       end
 
       context "when proposing a new EFI partition" do
+
```

Comment:
I'm surprised Rubocop didn't complain about this extra blank line. :-)

Reply:
Me too :-) Fixed.